common/djangoapps/util/milestones_helpers.py

```diff
@@ -376,9 +376,13 @@ def any_unfulfilled_milestones(course_id, user_id):
     """ Returns a boolean if user has any unfulfilled milestones """
     if not settings.FEATURES.get('MILESTONES_APP', False):
         return False
-    return bool(
-        get_course_milestones_fulfillment_paths(course_id, {"id": user_id})
-    )
+
+    from milestones import api as milestones_api
+    fulfillment_paths = milestones_api.get_course_milestones_fulfillment_paths(course_id, {'id': user_id})
+
```

Comment:
Just one comment, why not use the function in this file: ```get_course_milestones_fulfillment_paths``` ?
I see in some functions the ```milestones_api.get_course_.....``` is being used, and only in one function the ```get_course_milestones_fulfillment_paths``` in the file is being used. I do not see the difference in the result on one hand, and I do not see why ```from milestones import api as milestones_api``` has to be used many times?